addons/survey/static/src/js/survey_quick_access.js

```diff
@@ -24,8 +25,11 @@ publicWidget.registry.SurveyQuickAccessWidget = publicWidget.Widget.extend({
             if (!self.readonly) {
                 $(document).on('keypress', self._onKeyPress.bind(self));
             }
-
-            self.$('input').focus();
+            // Put the cursor after the input value
+            var session_code = self.$('input#session_code').val();
+            self.$('input#session_code').focus();
+            self.$('input#session_code').val('');
+            self.$('input#session_code').val(session_code);           
```

Comment:
Seems strange, we get the value from the input and then we reapply that value, and use a blank value in between?

(Also, camelCase).